docs/scss/_community.scss

```diff
@@ -0,0 +1,25 @@
+.ov-community {
+    display:flex;
+    flex-direction: column;
+    > *:not(:last-child) {
+    margin-bottom: var(--ov--spacer--md);
+    }
+    }
+
+.ov-masthead--community {
+	    background: url('/images/oVirt_Community_Header.png') left top no-repeat var(--ov--color-brand--100);
+    .ov-heading--lg {
+        max-width: 400px;
```

Comment:
It would probably be useful to do since a couple other sections do feature a masthead. I took the same class attributes from the homepage masthead but changed the image URL so it was specific for the community page.